WhateverGreen/kern_rad.cpp

```diff
@@ -183,7 +192,108 @@ void RAD::processKernel(KernelPatcher &patcher, DeviceInfo *info) {
 	}
 }
 
+uint32_t RAD::getMaxBrightnessFromDisplay() {
```

Comment:
This function always returns 0, make it void. Also rename to something like updateMaxBrightnessFromDisplay. get methods generally return the value not update some var.